src/main/scala/strawman/collection/immutable/HashMap.scala

```diff
@@ -28,8 +30,9 @@ import java.lang.{Integer, String, System}
 @SerialVersionUID(2L)
 sealed trait HashMap[K, +V]
   extends Map[K, V]
-     with MapOps[K, V, HashMap, HashMap[K, V]]
-     with Serializable {
+    with MapOps[K, V, HashMap, HashMap[K, V]]
+    with Buildable[(K, V), HashMap[K, V]]
```

Comment:
For `Map` it probably makes sense to extend `Buildable`, just like for `Set` (which I am working on at the moment)